app/assets/javascripts/services/miq_service.js

```diff
@@ -124,7 +124,7 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', '$log
 
     return serializedObj;
   };
-  
+
```

Comment:
Can you remove this blank line? Since there are no real changes here, let's not include this JS in this PR at all.